package.json

```diff
@@ -54,12 +54,10 @@
     "lodash": "^4",
     "minimatch": "^10",
     "mocha": "^10",
-    "playwright-core": "^1.48"
+    "playwright-core": "^1.49"
```

Comment:
shouldn't this be a separate PR? :trollface:

Reply:
oh wait, i'm the slow one today.... this isn't `playwright-ui-tests` 😖 
nvrmnd 🙈